lib/datadog/ci/ext/environment.rb

```diff
@@ -447,14 +454,25 @@ def extract_local_git
 
         def branch_or_tag(branch_or_tag)
           branch = tag = nil
-          if branch_or_tag.include?('tags/')
+          if !branch_or_tag.nil? && branch_or_tag.include?('tags/')
```

Comment:
Minor: This can be `branch_or_tag && branch_or_tag.include?(...)` since everything other than nil and false is considered true for an if ("truthy" is what we usually call it).